### PR TITLE
PLANET-3945 Convert Split two columns block to Gutenberg

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
[JIRA 3945](https://jira.greenpeace.org/browse/PLANET-3945)

### CODE REVIEW

File: react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    }
+
+    renderEdit() {
+      let tag_list   = this.props.tagsList.map((tag) => ({label: tag.name, value: tag.id}));
+      tag_list.unshift({label: '--Select Tag--', value: 0});
+
+      let issuepage_list = this.props.issuepageList.map((issue) => ({label: issue.title.raw, value: issue.id}));
+      issuepage_list.unshift({label: '--Select Issue--', value: 0});
+
+      const getIssueImageOrButton = (openEvent) => {
```
Comment (kirdia) [inline react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js]: Could you extract this to a class function to reduce the size of renderEdit()?

File: react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+                + {__('Select Image for Issue', 'p4ge')}
+              </Button>
+
+              <div>{__('(Optional)', 'p4ge')}</div>
+            </div>
+          );
+        }
+      };
+
+      const onCampaignImageFocalPointChange = (openEvent) => {
```
Comment (kirdia) [inline react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js]: same with getIssueImageOrButton, could you extract this to a class function to reduce the size of renderEdit()?

File: react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        if ( this.props.tag_image ) {
+
+          return (
+
+            <img
+              src={ this.props.tag_image_url }
+              onClick={ openEvent }
+              className="splittwocolumns__tag_imgs"
+              width={'250px'}
+              style={{padding: '10px 10px'}}
```
Comment (kirdia) [inline react-blocks/src/blocks/Splittwocolumns/Splittwocolumns.js]: can you extract this to a styling rule?

File: react-blocks/src/styles/blocks/SplitTwoColumns.scss [hunk 4]
```diff
@@ -0,0 +1,417 @@
+.split-two-column {
+  background-color: $black;
+  color: $white;
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  z-index: 2;
+  width: 100vw;
```
Comment (kirdia) [inline react-blocks/src/styles/blocks/SplitTwoColumns.scss]: this is breaking block render in the editor, might be better to disable it when the block is rendered inside the editor

**Comment by sagarsdeshmukh** [comment]
Hi @kirdia - I updated the PR, please have a look.

**Pull request merged**